infer/src/base/Config.ml

```diff
@@ -1633,6 +1633,13 @@ and force_integration =
        |> String.concat ~sep:", " ) )
 
 
+and fork_mode =
+  CLOpt.mk_bool ~long:"fork-mode"
```

Comment:
`--unix-fork`? (and so `--no-unix-fork`). I think it fits in better with our other option names and is a tad more descriptive. (I can change on our end if that's the only change)

Reply:
Done by Jules in a later commit. (And agreed, anyway!)